hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/EndpointBase.java

```diff
@@ -308,4 +308,14 @@ protected Map<String, String> getAuditParameters() {
     }
     return res;
   }
+
+  protected void auditWriteFailure(AuditAction action, Throwable ex) {
+    AUDIT.logWriteFailure(
+        buildAuditMessageForFailure(action, getAuditParameters(), ex));
+  }
+
+  protected void auditReadFailure(AuditAction action, Exception ex) {
+    AUDIT.logReadFailure(
+        buildAuditMessageForFailure(action, getAuditParameters(), ex));
+  }
```

Comment:
Extracting these is a good idea.  Did you intend to change only parts of the code which were touched anyway?  Would you be converting other call places in a future task?  Also, what about `audit(Read|Write)Success`?

Reply:
Yeah, I intended to keep the scope of this PR for the error logging. But actually we can extend it a little bit to success audits to make the code consistent. Good catch.